### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by hemajv**
Adding more content to refine the README
(addressing operate-first/support#959)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,35 @@
...
+## Services Monitored
+
+We have the [Open Data Hub](https://opendatahub.io/) applications deployed and running in a MOC [(Mass Open Cloud)](https://massopen.cloud/) cluster. Open Data Hub is an end-to-end AI/ML platform on top of OpenShift Container Platform which provides various tools for Data Scientists and Engineers.
+
+The components we currently have available are:
+* [JupyterHub](https://jupyterhub-opf-jupyterhub.apps.cnv.massopen.cloud/hub/login)
+
+## SLI and SLOs
+
+We have defined Service Level Indicators (SLI) and Service Level Objectives (SLO) for each of the ODH applications to help us monitor the availability of the service and improve its reliability.
```
Comment (anishasthana) [inline README.md]: We will eventually deploy non-ODH applications right? Should we change "for each of the ODH applications" to "SRE-managed Services" or something similar to that?

File: README.md [hunk 2]
```diff
@@ -1,3 +1,35 @@
...
+
+The `sli-slo` folder [here](https://github.com/operate-first/SRE/tree/master/sli-slo) lists out the SLI/SLO for each of the applications in separate markdown files. For eg, you can find the SLI/SLO defined for the `JupyterHub` application [here](https://github.com/operate-first/SRE/blob/master/sli-slo/jupyterhub.md).
+
+## Operational Tools
+
+TO DO
+
+## Runbooks
+
+For all the services being monitored, we have defined runbooks to provide information on how to administer, debug and effectively manage and troubleshoot common problems of a service.
```
Comment (anishasthana) [inline README.md]: ` administer, debug and effectively manage and troubleshoot common` -> ` administer, debug, and effectively troubleshoot common problems of a service`

File: README.md [hunk 3]
```diff
@@ -1,3 +1,35 @@
...
 
 This repository contains all the SRE (Site Reliability Engineering) principles and guidelines for managing the Operate-First services.
+
+## What is SRE?
+
+SRE is a software engineering approach to manage operations for systems, applications and services. We use software as a tool to manage systems, solve problems, and automate operations tasks.
+
+## Services Monitored
+
+We have the [Open Data Hub](https://opendatahub.io/) applications deployed and running in a MOC [(Mass Open Cloud)](https://massopen.cloud/) cluster. Open Data Hub is an end-to-end AI/ML platform on top of OpenShift Container Platform which provides various tools for Data Scientists and Engineers.
```
Comment (anishasthana) [inline README.md]: remove the first `the`

**Review by anishasthana — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (hemajv) [reply README.md]: fixed :+1:

**Review by anishasthana — APPROVED** [review]

**Review by HumairAK — APPROVED** [review]

**Pull request merged**